modules/patterns/pages/managing-multiple-versions.adoc

```diff
@@ -43,6 +43,11 @@ spec:
 
 To enable quickly creating multiple development streams, we must create a template for them.
 
+[NOTE]
```

Comment:
Do you want to make this more apparent like a warning?

Reply:
isn't it too strong? I'd keep it for more breaking changes

Reply:
How about CAUTION (I think that is one too)?